### ISSUE TITLE
Can "arguments label" be supported ?

### ISSUE THREAD

**Issue by NextSeason** [stylus/stylus#2253]
I don't know if stylus is keeping upgrade as I see the last commit is 7 month ago.
But I think I really need "arguments label" to sign the arguments I set. 
for example, function input in form.styl have two arguments `$color` and `width` and if I want to create different inputs, I must set `$color` everytime. This makes I must to find out the color which being used on input. So for me I think the order of `input` function's argument doesn't make sense, cuz I always need to change the size of `input` but not `color`.

So, to sum up, I think `labels` for arguments can solve this problem and I can use the function by not set all arguments : 

``` input( width 100% )```

**Comment by vendethiel** [comment]
We might not be looking at the same repo, the last commit I see was in july.

You want named arguments, right? [That's already supported](http://stylus-lang.com/docs/kwargs.html)

**Comment by NextSeason** [comment]
@vendethiel I am sorry about my mistake. But I really cannot use the `Keyword Arguments` like the way in the document.

I am using axis as my library and there is a `input` method in it which was defined as follow : 

```  input( $color = $aqua, $width = 250px ) ```

and if I call this function with ` input( $width : "100" )` I will get this error : 

>    1| .ns-iw222keu {
>    2|     width : 100%;
>    3|     position : relative;
>    4|     .input-line {
>    5|         input( width : "100%" );
> --------------------------------------^
>    6|         padding : 0;
>    7|         .ns-aside {
>    8|             width : auto;
> 
> expected ")", got ":"

Am I using the `Keyword Arguments` in a wrong way ?

Thanks for your reply.

**Comment by vendethiel** [comment]
Yes, you have an extra space, and you're using the wrong name.

You should be writing `input($width: 100%);` not `input( width : "100%" );`

**Comment by kizu** [comment]
Yep, there is an issue with whitespace. Closing this issue as it is already supported.
